### PR TITLE
ACC-1362 fix project and update to npm7

### PR DESCRIPTION
**Pull request by asugar13**
Updating npm to v7. Fixing the issue that was not letting the project run locally. Either way, I'm not sure this project is used anywhere so we may need to consider whether it's worth maintaining it. There is no config.yml for it.

[JIRA](https://financialtimes.atlassian.net/jira/software/c/projects/ACC/boards/1461?modal=detail&selectedIssue=ACC-1362)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,6 +34,14 @@
     "webpack-node-externals": "^1.7.2"
   },
   "engines": {
-    "node": "12.x"
+    "node": "12.x",
+    "npm": "7.x || 8.x"
+  },
+  "husky": {
```
Comment (asugar13) [inline package.json:40]: I think this husky bit has been done automatically. Let me know if I should revert it. The scripts seem to be running anyways.

File: package.json [hunk 2]
```diff
@@ -38,6 +35,16 @@
...
+    "hooks": {
+      "commit-msg": "node_modules/.bin/secret-squirrel-commitmsg",
+      "pre-commit": "node_modules/.bin/secret-squirrel",
+      "pre-push": "make verify -j3",
+      "prepare": "npx snyk protect || npx snyk protect -d || true",
+      "preinstall": "[ \"$INIT_CWD\" != \"$PWD\" ] || npm_config_yes=true npx check-engine"
```
Comment (polkwc) [inline package.json:47]: **Question**:  Why do you use this presinstall task as a hooks. In others migrations this is a task in the script section. Same with prepare.

File: .babelrc [hunk 3]
```diff
@@ -3,7 +3,7 @@
     [
       "env",
       {
-      	"targets": { "node": "12.x" }
+      	"targets": { "node": "12.22.9" }
```
Comment (polkwc) [inline .babelrc:6]: **Question**: why do you force a specific version? in the package.json file, you don't use a specific version.

**Review by polkwc — CHANGES_REQUESTED** [review]
**Issue (blocking):** Additional, you have to remove yarn.lock file and add the package-lock.json file.

File: package.json (hunk 1 above)
Comment (polkwc) [reply package.json:44]: ok

**Review by polkwc — APPROVED** [review]
well done

**Pull request merged**